### PR TITLE
Improve the Parallel Builds docs

### PR DESCRIPTION
**Pull request by harrietgrace**
Improves the [Parallel Builds](https://buildkite.com/docs/builds/parallel-builds) documentation.

- [x] Update screenshots

### CODE REVIEW

File: pages/builds/parallelizing_builds.md.erb [hunk 1]
```diff
@@ -36,11 +36,11 @@ To run the same step over all 5 of the agents we can set the parallelism field f
```
Comment (harrietgrace) [inline pages/builds/parallelizing_builds.md.erb]: Is there an example of a time where running the same step 5 times in parallel would be super useful?

File: pages/builds/parallelizing_builds.md.erb [hunk 2]
```diff
@@ -51,29 +51,28 @@ BUILDKITE_PARALLEL_JOB=0
...
-The following libraries have built-in support for the `BUILDKITE_PARALLEL_*` environment variables:
+The [Knapsack](https://github.com/ArturT/knapsack) ruby gem has built-in support for the `BUILDKITE_PARALLEL_JOB` and `BUILDKITE_PARALLEL_JOB_COUNT` environment variables:
 
-* [Knapsack](https://github.com/ArturT/knapsack)
-<br>A ruby gem for that will automatically divide your tests between parallel jobs, as well as making sure each job runs in comparable time. Supports RSpec, Cucumber and Minitest.
+Knapsack is a ruby gem for automatically dividing your tests between parallel jobs, as well as making sure each job runs in comparable time. It supports RSpec, Cucumber, and Minitest.
 
 ## Isolated jobs
 
 You can safely run multiple agents on a single machine, though you’ll still need to ensure your application’s code is able to be run in parallel. One convenient way of doing this is by using the agent’s [built in Docker and Docker Compose support](docker-containerized-builds) which will run each job in one or more completely isolated containers.
```
Comment (harrietgrace) [inline pages/builds/parallelizing_builds.md.erb]: When would it be unsafe? And why would running in isolated containers be better?

File: pages/builds/parallelizing_builds.md.erb [hunk 3]
```diff
@@ -51,29 +51,28 @@ BUILDKITE_PARALLEL_JOB=0
...
-* Using [agent prioritization](/docs/agent/prioritization) you’ll able to define a core set up agents that are assigned work first, and then lower priority agents which you may spin up and spin down on demand.
-* Using the [Agents API](/docs/api/agents) you’re able to fetch your agent’s `job` and `last_job_finished_at`, which allow you to see if they have been idle. You can then remotely shut down an agent process using the [agent stop API](/docs/api/agents#stop-an-agent) (passing `"force": false` to ensure it won’t cancel any running job).
+* Using the [Pipelines API](/docs/api/pipelines) you’re able to fetch the `scheduled_jobs_count`, `waiting_jobs_count` and `running_jobs_count` to get an overview of the number of jobs waiting to be run. Keeping track of this value, you can spin new agent machines up or down to accomodate the job load.
+* Using [agent prioritization](/docs/agent/prioritization) you’ll able to define which agents are assigned work first, as well as lower priority agents which you may spin up and down on demand.
```
Comment (harrietgrace) [inline pages/builds/parallelizing_builds.md.erb]: Do agents shut themselves down? Or do you need to spin them down again after using them?

File: pages/builds/parallelizing_builds.md.erb (hunk 1 above)
Comment (keithpitt) [inline pages/builds/parallelizing_builds.md.erb]: Totally! https://github.com/ArturT/knapsack#info-for-buildkitecom-users

That's how we parallelise our `rspec` tests.

File: pages/builds/parallelizing_builds.md.erb (hunk 2 above)
Comment (keithpitt) [inline pages/builds/parallelizing_builds.md.erb]: Never really :) Running multiple agents on the same machine is totes fine under all circumstances!

File: pages/builds/parallelizing_builds.md.erb (hunk 3 above)
Comment (keithpitt) [inline pages/builds/parallelizing_builds.md.erb]: If you hit the "Stop" button on the agent page, they shut themselves down. But in normal day to day work, the agents will stay connected until they're told to shutdown.

File: pages/builds/parallelizing_builds.md.erb (hunk 2 above)
Comment (harrietgrace) [inline pages/builds/parallelizing_builds.md.erb]: Cool. So why do we tell people about running isolated jobs?

Comment (keithpitt) [inline pages/builds/parallelizing_builds.md.erb]: Will explain in person :)

Comment (keithpitt) [inline pages/builds/parallelizing_builds.md.erb]:
```

test:
  adapter: postgresql
  encoding: unicode
  database: buildkite_test_<%= ENV['BUILDKITE_JOB_NUMBER'] %
  pool: 10
  username: postgres
  password:
```

File: pages/builds/parallelizing_builds.md.erb (hunk 1 above)
Comment (harrietgrace) [inline pages/builds/parallelizing_builds.md.erb]: Ah yeah. So why is that a thing people want to do? Why is it good to run the same tests at the same time on different things?

Comment (keithpitt) [inline pages/builds/parallelizing_builds.md.erb]: So knapsack will take your test suite, divide it into parts, and run each part on different agents. It's a super neat way of parallelising tests across multiple agents.

**Comment by toolmantim** [comment]
@harrietgrace alright, this is also ready to 🚀 I think! It had some really nice improvements, and I just fleshed out and updated a whole bunch of technical parts.

**Comment by harrietgrace** [comment]
Looks great! I can't approve this because it's my own PR, but consider the changes ~~ approved ~~ 🎉

**Comment by toolmantim** [comment]
😁👏🏼

**Pull request merged**